pkgs/applications/editors/notepadqq/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qtbase }:
+
+let
+  version = "v0.53.0";
+  sources = {
```

Comment:
I would prefer `fetchgit` with an explicit `fetchSubmodules = true` over fetching it manually and copying it